app/src/main/java/org/simple/clinic/medicalhistory/newentry/NewMedicalHistoryScreen.kt

```diff
@@ -164,8 +168,8 @@ class NewMedicalHistoryScreen : BaseScreen<
       router.pop()
     }
 
-    post {
-      hideKeyboard()
+    rootView.post {
+      rootView.hideKeyboard()
```

Comment:
Previously we were doing this to hide the keyboard when coming from the patient entry screen. Can we verify if it's still needed? or else we can remove this code as well.